src/main/java/org/jenkinsci/plugins/sqlplus/script/runner/SQLPlusRunnerBuilder.java

```diff
@@ -58,22 +59,22 @@
 	private String customSQLPath;
 
 	@DataBoundConstructor
-	public SQLPlusRunnerBuilder(String credentialsId, String instance, String scriptType, String script,
+	public SQLPlusRunnerBuilder(String credentialsId, String user, String password, String instance, String scriptType, String script,
 			String scriptContent) {
 		this.credentialsId = credentialsId;
-		this.user = null;
-		this.password = null;
+		this.user = user;
+		this.password = password;
```

Comment:
Need to confirm this doesn't bring any security issues, by storing a password in plain-text in the object (which will be serialized/persisted to XML)

Reply:
I will merge now and do some tests about it